dojo/tools/qualys/parser.py

```diff
@@ -185,6 +220,8 @@ def parse_finding(host, tree):
         finding.mitigated = _temp['mitigation_date']
         finding.is_Mitigated = _temp['mitigated']
         finding.active = _temp['active']
+        if _temp.get('CVSS_vector') is not None:
+            finding.cvssv3 = _temp.get('CVSS_vector')
```

Comment:
We are mixing vector v2 and v3 here :/ 
Sadly for now the data model doesn't support v2 so I'm not a fan of putting wrong data here

Reply:
v2 would not match the regex and therefore not used here

Reply:
if the vector comes from v2 is set to None now

Reply:
ok